_sources/Functions/thinkcspyExercises.rst

```diff
@@ -551,3 +551,13 @@ Exercises
             .. disqus::
                 :shortname: interactivepython
                 :identifier: db5d8808bf5749579718bdd2088b539f
+
```

Comment:
I missed that!  I think I fixed a couple.  It looks like the lists Exercises still have a number of problems with that issue that I did not cause, I think.